script.js

```diff
@@ -1,4 +1,308 @@
+var playerOneRoll = [];
+var playerTwoRoll = [];
+var playerOneScore = 0;
+var playerTwoScore = 0;
+var playerOneChoice;
+var playerTwoChoice;
+var gameModes = [
+  "normal-highest",
+  "normal-lowest",
+  "auto-highest",
+  "auto-lowest",
+];
+var currentGameMode = "game start";
+var currentPlayer = "Player One";
+var numDice = 2; //number of dice defaults at 2 in case player did not choose any, will change on game modes
+var winner = "";
+//assign who is playing
+var playerOneIsPlaying = true;
+var playerTwoIsPlaying = false;
+var compareScores = false;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (currentGameMode == "game start") {
+    currentGameMode = "choose number of dice";
+    console.log(currentGameMode);
+    return `How many dice you want to play?`;
+  }
+
+  if (currentGameMode == "choose number of dice") {
+    numDice = input; // variable number of dice
+    currentGameMode = "choose game mode";
+    return "Please choose Game Modes: <br>normal-highest<br> normal-lowest <br>auto-highest <br>auto-lowest";
+  }
+  if (currentGameMode == "choose game mode") {
+    //Loops thru the gameModes and output the instruction for the chosen game mode
+    var counter = 0;
+    while (counter < gameModes.length) {
```

Comment:
nice how you looped the array to find the gameMode and output instructions! good job